src/pages/hotel/booking/book-header/BookHeader.tsx

```diff
@@ -17,12 +17,12 @@ function BookHeader() {
       <div className="book-items display-content">
         <div className="items">
           <div className="book-indicator progress-divider">1</div>
-          <div className="title display-content">Customer Information</div>
+          <div className="title display-content">Customer Info</div>

```

Comment:
Instead of changing the texts directly for all, may be we can change the texts for small screens only, which have the readability issues.

Reply:
ok